### DISCUSSION TITLE
4 days Of Freestyle Silly

### DISCUSSION THREAD

**Discussion by Diabeticliberty** [NightscoutFoundation/xDrip#2978] [General]
In the last 4 days I have burned about half a dozen Libre 2 sensors. They would not function through Xdrip+ They would not function through Librelink which is Abbott's own stock app. I know I am not alone in this as the dozen or so other diabetics with whom I am in regular contact have all encountered pretty similar scenarios. In a nutshell after doing the Libre warm up scan the sensors just gave up. They certainly would not function with Xdrip+. To any of you out there experiencing similar issues  the Librelink app update which Abbott released today on Google Play Store seems to have fixed things at least in part on their side of things. Fitted a new sensor tonight and Xdrip+ scanned as I dearly needed it to after the warm up period. You never realise how much you value Xdrip+ until you lose it. Time for me to offer my trademark thanks to all parties involved in its continued development and success. You have enriched the lives of so many diabetics. Probably far more than any of you realise.

**Comment by 3DRZ** [comment]
Same here. In the UK they released Libre Link in a new version wich turns the libre 2 in a cgm.
So the values will appear on the app without scanning. Hope they are releasing it also in other countries. As soon as this happens I whipe xdrip from my phone - had nothing but problems with xdrip (like burnt sensors and crazy vibrations on connection loss, etc)

**Reply by Diabeticliberty** [reply]
I am really sorry that you have had negative experiences with Xdrip+. It has been fantastic for me and converted Libre which prior to finding Xdrip+ was extremely patchy into something I really can trust and rely on. As regards the Librelink update? I am based in the UK and I think that once it becomes available to you then there is every likelihood that you will be underwhelmed. I find that where Xdrip+ wins for me every single time is the fact that it can be calibrated to marry up with fingerprick blood test results. The Libre update still precludes users from calibrating and while they insist that Librelink does not require calibration I can tell you from personal experience that it really does. I was contacted last night by someone who had just installed the new update and Librelink was a full 7 points (mmols) different from her actual fingerprick test. For a comparison here is one of mine this morning

![Xdrip+ And Accuchek](https://github.com/NightscoutFoundation/xDrip/assets/44977244/bfd669bb-5d36-4d1e-b874-6934c3c2bd9b)

In addition to this Librelink has no facility for a widget on a mobile phone front screen. There are a number of apps on Google Play Store which suggest that they allow a Librelink widget creation. None of them work and Abbott have no intentions of including one in further Librelink updates. 

A further major benefit for me for Xdrip+ is smartwatch integration which is something else not available through Librelink. The ability when woken up hypoglycaemic at 3.00am in a morning and being able to immediately kill the alarm from my wrist without waking my fiancee is worth a solid gold clock. While there is the facility to be able to do this with the Libre link CGM alarms it is extremely limited. Rather shockingly Librelink disabled all of the high and low glucose alarms that I had set since the 'app upgrade' I went hypoglycaemic and had no alarms at all. This of course did wake my fiancee and probably my next door neighbour too. 

Librelink gave me so many lost connection alerts that in the end I personally disabled that function.

I do not suggest that Xdrip+ is perfect. Nothing in life is. I will however, extremely confidently stress that it is a thousand miles ahead of the Librelink app and I to say I am grateful for it would be an enormous understatement.

**Comment by Diabeticliberty** [comment]
Xdrip+ continues to display results which are extremely close to fingerprick blood testing. How close? Well that's a toughy, actually it isn't that tough. Results are identical. I rarely if ever get this from Librelink.

![20230805_091900](https://github.com/NightscoutFoundation/xDrip/assets/44977244/d5220269-92a5-4eb5-a7ac-a23123d557f4)
